code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -350,15 +325,21 @@
 	S.key = T.key
 	if(U)
 		S.faction |= "\ref[U]" //Add the master as a faction, allowing inter-mob cooperation
+		if(!iscultist(U))
+			S.mind.store_memory("<b>Serve [U.real_name], your creator.</b>")

```

Comment:
Cultist-created shades don't get a note to serve the cult? Or is it handled by `.special_role = SPECIAL_ROLE_CULTIST`?

Reply:
They don't get notes to serve the cult, they get notes with the cult objectives though. SPECIAL_ROLE_CULTIST is for logging as Antag.

Reply:
Well I added a note to serve the cult because why not.